src/lib/holocene/monaco/markdown.svelte

```diff
@@ -40,13 +48,13 @@
   const templatedContent = replaceTemplate(content);
 </script>
 
-<section class="h-full w-full" in:fade={{ duration: 1000 }}>
+<section class={`h-full w-full ${className}`} in:fade={{ duration: 1000 }}>
```

Comment:
nit: using template literal interpolation here isn't necessary

```suggestion
<section class="h-full w-full {className}" in:fade={{ duration: 1000 }}>
```
In fact, we should probably use `twMerge` for stuff like this instead.